### PR TITLE
Use system dictionary for transfer states

### PR DESCRIPTION
**Pull request by MatthewPhinney**

### CODE REVIEW

**Comment by naoitoi** [comment]
How about defining the states in -shared, so that all the ILP components can use the same dictionary?
Inter-component consistency ...

**Comment by MatthewPhinney** [comment]
I'm open to that idea. You could argue that we already do define the states in shared: https://github.com/interledger/five-bells-shared/blob/master/schemas/TransferState.json. We could move the transferState file into shared/lib.

**Comment by MatthewPhinney** [comment]
@naoitoi updated to use the library in five-bells-shared@11.1.0.

File: src/lib/transferExpiryMonitor.js [hunk 1]
```diff
@@ -65,9 +68,8 @@ class TransferExpiryMonitor {
         log.debug('transfer ' + transfer.id +
           ' will expire in ' + expiry.diff(now, 'milliseconds') + 'ms')
       }
-    } else if (transfer.state === 'executed' ||
-      transfer.state === 'rejected' ||
-      transfer.state === 'failed') {
+    } else if (transfer.state === transferStates.TRANSFER_STATE_EXECUTED ||
+      transfer.state === transferStates.TRANSFER_STATE_REJECTED) {
```
Comment (naoitoi) [inline src/lib/transferExpiryMonitor.js]: Is it OK not to check for 'failed'?

**Comment by naoitoi** [comment]
LGTM other than:
- Changing the way how we require(), as discussed offline, and
- Failing unit test

File: src/lib/transferExpiryMonitor.js (hunk 1 above)
Comment (MatthewPhinney) [inline src/lib/transferExpiryMonitor.js]: `failed` is no longer a valid transfer state, as per the schema: https://github.com/interledger/five-bells-shared/blob/master/schemas/TransferState.json.

**Comment by MatthewPhinney** [comment]
I updated the requires to be consistent with `five-bells-shared@11.1.1`: `const transferDictionary = require('five-bells-shared').TransferStateDictionary`

**Comment by naoitoi** [comment]
LGTM, thanks!

**Pull request merged**